worker/mozillaGradingWorker/ciphersuites.go

```diff
@@ -0,0 +1,1295 @@
+package mozillaGradingWorker
+
+//OpenSSLCiphersuites contains all the ciphersuites recognised by OpenSSL
+var OpenSSLCiphersuites = `{
```

Comment:
Excellent! Then let's make it a go generate task that runs when we compile
the scanner so it's always up to date.

On Jul 15, 2016 08:22, "Dimitris Bachtis" <notifications@github.com> wrote:

> In worker/mozillaGradingWorker/ciphersuites.go
> <https://github.com/mozilla/tls-observatory/pull/125#discussion_r70963257>
> :
>
> > @@ -0,0 +1,1295 @@
> > +package mozillaGradingWorker
> > +
> > +//OpenSSLCiphersuites contains all the ciphersuites recognised by OpenSSL
> > +var OpenSSLCiphersuites = `{
>
> I generated the file by running the OpenSSL output through Go, so it is
> even more silly that I did not include it 😶
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mozilla/tls-observatory/pull/125/files/e95a1ebf30930b63af3799e12d7f2c0dba55e968#r70963257>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAZXgZVcZlnkLsptV-xtEcBaH1DAOfRuks5qV3uVgaJpZM4JNMIS>
> .
>